tracker/utils.py

```diff
@@ -69,21 +69,20 @@ def check_issue_assignment_events(issue: dict) -> dict:
         response = requests.get(events_url, headers=HEADERS)
         response.raise_for_status()
 
-        if response.ok:
-            events = response.json()
-            assignment_info = {
-                "assignee": None,
-                "assigned_at": None,
-            }
-
-            for event in events:
-                if event.get("event") == "assigned":
-                    assignment_info["assignee"] = event.get("assignee", dict()).get(
-                        "login", str()
-                    )
-                    assignment_info["assigned_at"] = event.get("created_at", str())
-
-            return assignment_info
+        events = response.json()
+        assignment_info = {
```

Comment:
Use defaultdict if you need to return here empty dict
`event.get("assignee", dict())`